docs/learn/learn-proxies.md

```diff
@@ -106,9 +107,9 @@ Learn more about anonymous proxies from our
 ### Time Delayed Proxies
 
 We can add an additional layer of security to proxies by giving them a delay time. The delay will be
-quantified in number of blocks (blockNumber). Polkadot and Kusama both have
-{{ block_target_in_seconds }} second blocks, hence a delay value of 10 will mean 10 blocks which
-will equal 1 minute of delay. The proxy will announce it's intended action and wait for the number
+quantified in number of blocks (blockNumber). Polkadot and Kusama both have 6 
```

Comment:
I was having trouble finding how to retrieve this value on-chain and asked about how to calculate it in `W3F Dumb questions and answers`.

Reply:
Just for reference 

This was the solution provided by Philipp:
```
Iirc the blockheader metadata has a timestamp
```

joe elaborated with:
```
timestamp.now.at(<block_hash>)

Or if you want it from a block (instead of chainstate) then you can just match on the set timestamp extrinsic

{
  "method":{
    "pallet":"timestamp","method":"set"
  },
  "signature":null,
  "nonce":null,
  "args":{"now":"1658382702006"}
}

I don't think the block time is in chainstate anywhere (perhaps under some BABE config, but I don't have time to look at the moment). But it's easy to get empirically from (time(B) - time(A)) / (B - A)
```